include/constants/pokemon.h

```diff
@@ -435,129 +435,129 @@
 
 #ifndef PM_ASM
 typedef enum {
-    EGG               = 0,
-    EVENT             = 0,
-    HATCHED           = 0,
-    PAL_PARK          = 0,
-    TALL_GRASS        = 2,
+    EGG = 0,
+    EVENT = 0,
+    HATCHED = 0,
+    PAL_PARK = 0,
+    TALL_GRASS = 2,
```

Comment:
Any way that we can keep the alignment on enum/do it manually or will clang-format override them all the time?

Reply:
it's a WIP issue in clang-format, if someone was to implement it, we could use clang-format 20 git to have it
the alternative is to go to each enum and manually align it with a clang-format off but that'd be a lot of work